library/engine.gni

```diff
@@ -22,3 +22,13 @@ if (is_linux) {
     "$engine_download_dir/flutter_embedder.h",
   ]
 }
+
+if(is_win) {
+  engine_files = [
+    "$engine_download_dir/flutter_engine.dll",
+    "$engine_download_dir/flutter_embedder.h",
```

Comment:
I think it'll be clearer to move this up a line, so that all the dll-related files are grouped. (You can flip the Linux order for consistency.)